### DISCUSSION TITLE
Virtual Machine does not link to the IP address

### DISCUSSION THREAD

**Discussion by encorer** [netbox-community/netbox#6943] [Q&A]
When I try edit IP address (IPAM -> IP Adresses), and add virtual machine (Interface Assignment -> Virtual Machine), i got error:

Exception while resolving variable 'disabled' in template 'django/forms/widgets/select.html'.
Traceback (most recent call last):
  File "/opt/netbox-2.11.10/venv/lib/python3.8/site-packages/django/template/base.py", line 829, in _resolve_lookup
    current = current[bit]
TypeError: string indices must be integers

Tried to re-install cleanly, got the same. What can be done? Thanks.

**Comment by jeremystretch** [comment]
My guess is that something's gone wrong with your installation, as that's a very unusual exception. Are you able to reproduce the issue on the [pubic demo instance](https://demo.netbox.dev/)?

**Reply by encorer** [reply]
Yep, the same thing, from the gui side it looks as if you are editing ip adress in IPAM, assigning a virtual machine, click "update", the green inscription says that "Modified IP address 10.0.0.10/16", but if you look, then the virtual machine for the address was not assigned. And at least "Site" and "Region" in "NAT IP (Inside)" of ip address, in the same page, can't assign too.

**Reply by jeremystretch** [reply]
In that case please open a [bug report](https://github.com/netbox-community/netbox/issues/new?assignees=&labels=type%3A+bug&template=bug_report.yaml) providing the exact steps someone can follow to reproduce the error you're seeing.

**Reply by encorer** [reply]
thanks, created

**Reply by jeremystretch** [reply]
I'm a bit lost here. Your original post above cites an exception (the server error), but the bug report you opened (#6946) does not.

**Reply by encorer** [reply]
I thought that you yourself would get to the error, but I can add it
